### PR TITLE
Create EnemEditionSelected to handle Enem year

### PR DESCRIPTION
**Pull request by raulbcs**
## Description
- Create EnemEditionSelected class and tests
- Modify view
- Modify tests

### CODE REVIEW

**Review by marcusesa — COMMENTED** [review]
@raulbcs take care with copy/paste, looking for `census` you can see a couple of things.

File: src/AppBundle/Enem/EnemFilter.php [hunk 1]
```diff
@@ -19,9 +20,12 @@ class EnemFilter
         2009
```
Comment (marcusesa) [inline src/AppBundle/Enem/EnemFilter.php:20]: Shouldn't it be the same range in `EnemFilter` and `EnemEditionSelected`?

**Review by marcusesa — COMMENTED** [review]
Sorry, I have a new question.

File: src/AppBundle/Enem/EnemFilter.php (hunk 1 above)
Comment (raulbcs) [reply src/AppBundle/Enem/EnemFilter.php:20]: Yes, they should. I'll fix.

**Review by marcusesa — APPROVED** [review]
Let's go to deploy 🚀

**Pull request merged**